kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue

```diff
@@ -103,6 +103,8 @@
       learners: 'Learner reports',
       name: 'Name',
       group: 'Group',
+      channelIconColumnHeader: 'Channel icon',
+      userIconColumnHeader: 'User icon',
```

Comment:
@radinamatic in situations like this:
https://github.com/DXCanas/kolibri/blob/33edc0261de6d7e5b94b5fea1a5dc942afadbac9/kolibri/plugins/coach/assets/src/views/class-list-page/index.vue#L13

Where there is a column entirely dedicating to displaying icons, is it helpful/appropriate to add an invisible column header to label that column?

Reply:
Kudos for trying to thoroughly think this through from the user perspective @david! 
I cannot run this locally to test in place (hopefully my new testing machine will arrive tomorrow, and I can be back on track), but from the mockups I infer that the icons in the column are all the same, right? All users, or classes, etc.. In that case I would say it's safe to assume that the icons have mostly decorative function and we can make them invisible, which renders the invisible headers for those columns unnecessary :slightly_smiling_face:

Reply:
I don't think making the icons themselves invisible is an option on the table :/ Did you mean invisible to screenreaders?

Reply:
Yes, sorry for being imprecise, that's exactly what I meant!

Reply:
You know the best way to go about making them invisible to screen readers? I'm unclear on how they go through tables